cmd/dep/status.go

```diff
@@ -150,14 +149,15 @@ func (out *jsonOutput) MissingLine(ms *MissingStatus) {
 }
 
 func (out *jsonOutput) MissingFooter() {
-	json.NewEncoder(out.w).Encode(out.missing)
+	b, _ := json.Marshal(out.missing)
+	out.logger.Print(b)
 }
 
 type dotOutput struct {
-	w io.Writer
-	o string
-	g *graphviz
-	p *dep.Project
+	logger *log.Logger
```

Comment:
I think it's probably preferable to have this remain as an `io.Writer`, then map that to the `*log.Logger` in some intermediate step.